oidv6_to_voc/__init__.py

```diff
@@ -42,23 +42,22 @@ def convert_annfile(annfile: str, desc: str, imgd: str, outd: str):
         p.name.rstrip('.jpg')
         for p in imgp.rglob('*.jpg') if p.is_file()
     }
-    annl = []
 
+    annl: dict[str,  List[AnnotationRow]] = dict()
```

Comment:
Python <= 3.7 may not support `dict[...]` typing annotation. Use `Dict` from package `typing` instead.

Reply:
Done